src/noise.cpp

```diff
@@ -166,6 +167,7 @@ float noise2d(int x, int y, s32 seed)
 {
 	unsigned int n = (NOISE_MAGIC_X * x + NOISE_MAGIC_Y * y
 			+ NOISE_MAGIC_SEED * seed) & 0x7fffffff;
+	// Right bit shift on an unsigned type zero-fills.
```

Comment:
Obviously, I don't think this is worth writing down.

Reply:
This is a token of my inexperience as a developer. Thank you for pointing this out, I'll fix it ASAP!